specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+          "type": "string"
+        },
+        "operation": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationResult": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets action name",
+          "type": "string"
+        },
+        "isDataAction": {
+          "description": "Gets or sets a value indicating whether it is a data plane action",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "Gets or sets Display properties"
+        },
+        "properties": {
+          "description": "Gets or sets properties",
+          "type": "string"
+        },
+        "origin": {
+          "description": "Gets or sets origin",
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "Display class",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Gets or sets RP friendly name",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Gets or sets Resource type friendly name",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Gets or sets operation name",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or sets operation description",
+          "type": "string"
+        }
+      }
+    },
+    "LockboxRequestResponse": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LockboxRequestResponseProperties"
+        }
+      }
+    },
+    "LockboxRequestResponseProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "justification": {
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "Initializing",
+            "Pending",
+            "Completing",
+            "Completed",
+            "Unknown"
+          ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "duration": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "resourceId": {
+          "type": "string"
+        },
+        "resourceType": {
+          "type": "string"
+        },
+        "supportRequest": {
+          "type": "string"
+        },
+        "supportCaseUrl": {
+          "type": "string"
+        },
+        "subscriptionId": {
```

Comment:
subscriptionId can be inferred from resourceid.

Reply:
We will use the resourceId if it contains the necessary info.

Reply:
After reviewing, this resourceId is currently not the ARM resourceId, but instead a friendly resource id that we generate on our end, we will look into making this the ARM resource id for consistency's sake.